test/integration/controller/issuer/issuer_suite_test.go

```diff
@@ -47,19 +47,34 @@ var (
 	ctx context.Context
 	log logr.Logger
 
-	restConfig     *rest.Config
-	testEnv        *envtest.Environment
-	testClient     client.Client
-	kubeconfigFile string
+	restConfig           *rest.Config
+	testEnv              *envtest.Environment
+	testClient           client.Client
+	acmeDirectoryAddress string
+	kubeconfigFile       string
 
 	scheme *runtime.Scheme
 )
 
 var _ = BeforeSuite(func() {
-
 	logf.SetLogger(logger.MustNewZapLogger(logger.DebugLevel, logger.FormatJSON, zap.WriteTo(GinkgoWriter)))
 	log = logf.Log.WithName(testID)
 
+	By("Start Pebble ACME server")
+	certificatePath, directoryAddress, err := testutils.RunPebble(log.WithName("pebble"))
+	Expect(err).NotTo(HaveOccurred())
+	acmeDirectoryAddress = directoryAddress
```

Comment:
Sets the package scoped variable that tests will use to create `Issuer` resources.